cmd/mr_create.go

```diff
@@ -124,14 +124,15 @@ func runMRCreate(cmd *cobra.Command, args []string) {
 	if err != nil {
 		log.Fatal(err)
 	}
-	targetBranch := "master"
+
+	targetBranch := lab.GetDefaultBranch(targetProjectName)
```

Comment:
We probably don't need to add a function for this, we already have the target projects so:
```suggestion
	targetBranch := targetProject.DefaultBranch
```

Reply:
I'll make this change now and retest.